tfe/data_source_workspace_ids.go

```diff
@@ -56,7 +56,7 @@ func includedByName(names map[string]bool, workspaceName string) bool {
 		case len(name) == 0:
 			continue
 		case !strings.HasPrefix(name, "*") && !strings.HasSuffix(name, "*"):
-			if strings.Contains(workspaceName, name) {
+			if name == workspaceName {
```

Comment:
👍 Nice catch. Wondering how the behavior of `strings.Contains(workspaceName, name)` creates a bug, since they both perform a string comparison?

Reply:
It's because `Contains` doesn't match the whole string -- `strings.Contains("partial", "art")` will return true, but `"partial" == "art"` is false.